src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -164,6 +164,22 @@
           </div>
         </div>
 
+        <!-- System User -->
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
+            <div class="custom-control custom-checkbox">
+              <input class="custom-control-input"
+                     id="system"
+                     type="checkbox"
+                     formControlName="system">
+              <label class="custom-control-label"
+                     for="system"
+                     i18n>System</label>
```

Comment:
? to be more specific with it?
```suggestion
                     i18n>System user</label>
```

Reply:
Since this is the user form, I tried to avoid redundancy.

From the cli you would set the `--system` flag

Reply:
yep, but we should be more clear on what the `system` is in one go even if CLI is not.